stix_shifter_utils/utils/entry_point_base.py

```diff
@@ -144,21 +144,52 @@ def get_results_translator(self, dialect=None):
         return self.__dialect_to_results_translator[self.__dialect_default]
 
     @translation
-    def transform_query(self, dialect, data, antlr_parsing, options, mapping=None):
+    def transform_query(self, dialect, data, antlr_parsing):
         data_model_mapper = self.get_data_mapper(dialect)
         translator = self.get_query_translator(dialect)
-        return translator.transform_query(data, antlr_parsing, data_model_mapper, options, mapping)
+        return translator.transform_query(data, antlr_parsing, data_model_mapper)
 
     @translation
-    def translate_results(self, data_source, data, options):
+    def translate_results(self, data_source, data):
         translator = self.get_results_translator()
         return translator.translate_results(data_source, data)
 
     @translation
     def get_dialects(self):
         return self.__dialects_visible
 
-    def setup_transmission_basic(self, connector):
+    def setup_transmission_simple(self, connection, configuration):
```

Comment:
The name of this method is a little confusing. Simple as opposed to what? It would be nice to have a comment explaining what it's doing.